index.html

```diff
@@ -19,9 +19,12 @@
     Good luck and Happy Coding!
 
     - The freeCodeCamp Team
+    
 
     -->
  
+    <!-- <header></header> -->
```

Comment:
I think it would be better to avoid commented code in your HTML

as it makes your code not clean

Reply:
:+1: